src/common/hooks/useRemoveFromPackage/useRemoveFromPackage.js

```diff
@@ -0,0 +1,49 @@
+import { useCallback } from 'react';
+
+import { useModalToggle, useShowCallout } from '@folio/stripes-acq-components';
+
+import { useTitleMutation } from '../useTitleMutation';
+
+const ERROR_CODES = {
+  emptyHoldings: 'existingHoldingsForDeleteConfirmation',
+};
+
+export function useRemoveFromPackage({ id, onSuccess }) {
+  const [isRemoveFromPackageOpen, toggleRemoveFromPackageModal] = useModalToggle();
+  const [isRemoveHoldingsOpen, toggleRemoveHoldingsModal] = useModalToggle();
+  const showCallout = useShowCallout();
+  const { deleteTitle } = useTitleMutation();
+
+  const onConfirmRemoveFromPackage = useCallback(async (searchParams = {}) => {
+    try {
+      await deleteTitle({ id, searchParams });
+      onSuccess();
+      showCallout({ messageId: 'ui-receiving.title.confirmationModal.removeFromPackage.success' });
+    } catch (error) {
+      const { errors } = await error.response.json();
+      const { message, code } = errors[0];
```

Comment:
Could you please use `ResponseErrorsContainer` for that purpose? It's more safer and prevent exceptions if there is an error during converting a response to JSON ([ref](https://github.com/folio-org/stripes-acq-components/blob/6180443bf37966a6408e35237bab539f6a481c62/lib/utils/errorHandling/ResponseErrorsContainer.js))

```
const { handler } = await ResponseErrorsContainer.create(error?.response);
const { message, code } = handler.getError();
```

The "Strategy" pattern is particularly useful for handling error codes, and this utility was created for that purpose. You can find an example of how it works [here](https://github.com/folio-org/ui-orders/blob/f50bcd4d0b96c6ac9cfa44d71d803256e72696c8/src/components/Utils/order/showUpdateOrderError.js#L5).